__tests__/app.test.js

```diff
@@ -113,3 +113,54 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: responds with an array of comments for the given article_id with the required properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
You could use nested destructuring here {body: {comments}} so that you don't need to do line 123